q2_phylogeny/plugin_setup.py

```diff
@@ -60,6 +62,33 @@
     citations=[citations['price2010fasttree']]
 )
 
+plugin.methods.register_function(
+    function=q2_phylogeny.raxml,
+    inputs={
+            'alignment': FeatureData[AlignedSequence]},
+    parameters={
+            'seed': Int,
+            'n_threads': Int % Range(1, None),
+            'substitution_model': Str % Choices(_RAXML_MODEL_OPT)},
+    outputs=[('tree', Phylogeny[Unrooted])],
+    input_descriptions={
+        'alignment': ('Aligned sequences to be used for phylogenetic '
+                      'reconstruction.'),
+    },
+    parameter_descriptions={
+        'n_threads': ('The number of threads. Using more than one thread '
```

Comment:
How about: `The number of threads to use for multithreaded processing. Using more than...`?